apps/coscrad-frontend/src/components/About/About.spec.tsx

```diff
@@ -1,10 +1,26 @@
+import { screen } from '@testing-library/react';
+import { getDummyConfigurableContent } from '../../app/App.spec';
 import { renderWithProviders } from '../../utils/test-utils';
 
-import About from './About';
+import { About } from './About';
 
 describe('About', () => {
     it('should render successfully', () => {
         const { baseElement } = renderWithProviders(<About />);
         expect(baseElement).toBeTruthy();
     });
+
+    it('should load configurable content', () => {
+        const dummyConfigurableContent = getDummyConfigurableContent();
```

Comment:
I wonder if we want to change the API to 
```ts
buildDummyConfigurableContent(overrides: Partial<ConfigurableContant>)
```
Any thoughts?

Reply:
That would mean being more explicit about overriding props in the testing?  Sounds good to me.

Reply:
We will address this when we have more tests of this type.